src/services/delayModuleService.ts

```diff
@@ -1,12 +1,16 @@
 import { StaticJsonRpcProvider } from '@ethersproject/providers'
 import { Contract, ethers, utils } from 'ethers'
+import { _TypedDataEncoder, getAddress } from 'ethers/lib/utils'
 import { MetaTransaction } from 'ethers-multisend'
 
 import delayModuleABI from '../contracts/abis/Delay.json'
 import delayModuleFactoryABI from '../contracts/abis/DelayFactory.json'
+import { RecoveryParamsResponse } from '../wallet'
+import { AddressNotGuardianError } from '../wallet/errors'
 
 const DelayModule = new utils.Interface(delayModuleABI)
 const DelayModuleFactory = new utils.Interface(delayModuleFactoryABI)
+const SENTINEL = '0x0000000000000000000000000000000000000001'
```

Comment:
We already have that value as SAFE_SENTINEL_OWNERS in constants (just need to hexPad as we did)